tool_services/gate/TinyJSON/convert.c

```diff
@@ -487,10 +487,13 @@ void Convert_JsonToMsg(service_t *service, uint16_t id, luos_type_t type, const
         Luos_SendMsg(service, msg);
     }
     // reinit
-    if (json_getProperty(jobj, "reinit") != NULL)
+    item = json_getProperty(jobj, "reinit");
+    if (item != NULL)
```

Comment:
You should do that only if this property contain a Real value using `if (json_getType(item) == JSON_REAL)`
Then make an `else if (item != NULL)` with the previous behavior.
This way the previous behavior still the same and you can send reinit information if you want to.